src/NUnitFramework/framework/nunit.framework-4.5.csproj

```diff
@@ -123,6 +123,7 @@
     <Compile Include="Attributes\DatapointAttribute.cs" />
     <Compile Include="Attributes\DatapointsAttribute.cs" />
     <Compile Include="Attributes\DatapointSourceAttribute.cs" />
+    <Compile Include="Attributes\DataSourceAttribute.cs" />
```

Comment:
Oh! I suppose this is why we are only failing in the .NET 4.5 build - the code is not in the other builds yet!